hack/tools/install-cert-mgr.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
```

Comment:
Could we extract the cert manager version into a make version so that we can twist that knob from there?

I also wonder if it isn't just better to inline this `kubectl apply` line in the Makefile, especially if my hunch about the following comment is true (that we don't need to actually wait for cert-manager to be ready)

Reply:
I put it the line in the Makefile first then I realized that the wait is necessary so I created a script.  I can make changes to pass the version as an argument for the script.